examples/keras_io/vision/image_classifier.py

```diff
@@ -0,0 +1,679 @@
+# -*- coding: utf-8 -*-
+"""Classification in Keras-CV
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1qvQbaRuuMKJzp7cYgEQ7lY6kvBzglIqj
+
+# Classification with KerasCV
+
+**Author:** [lukewood](https://lukewood.xyz)<br>
+**Date created:** 03/28/2023<br>
+**Last modified:** 03/28/2023<br>
+**Description:** Use KerasCV to train powerful image classifiers.
+
+Classification is the process of predicting a categorical label for a given
+input image.
+While classification is a relatively straightforward computer vision task,
+modern approaches still are built of several complex components.
+Luckily, KerasCV provides APIs to construct commonly used components.
+
+This guide demonstrates KerasCV's modular approach to solving image
+classification problems at three levels of complexity:
+
+- Inference with a pretrained classifier
+- Fine-tuning a pretrained backbone
+- Training a image classifier from scratch
+
+We use Professor Keras, the official Keras mascot, as a
+visual reference for the complexity of the material:
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_evolution.png)
+
+## Multi-Backend Support
+
+Keras-CV's `ImageClassifier` model supports several backends like Jax, PyTorch, and TensorFlow with the help of `keras_core`. To enable multi-backend support in Keras-CV, set the `KERAS_CV_MULTI_BACKEND` environment variable. We can then switch between different backends by setting the `KERAS_BACKEND` environment variable. Currently, `tensorflow`, `jax`, and `torch` are supported.
```

Comment:
- Keep lines under 80 chars
- Refer to KerasCV as "KerasCV"

Reply:
Done in #608 with the exception of a few links that can't be wrapped to fit in 80 chars character limit.